### PR TITLE
fix: remove tinro from Tab and ui

### LINKED ISSUE #7235: remove tinro from ui component library
### Is your enhancement related to a problem? Please describe

Tab component brought the tinro component into the ui library

but the UI library should be agnostic on the routing system used by the framework.

### Describe the solution you'd like

Need to refactor the tab component and remove tinro from package.json

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by lstocchi**
### What does this PR do?

It removes the tinro dependency from the ui package

### Screenshot / video of UI

N/A

### What issues does this PR fix or reference?

it resolves #7235 

### How to test this PR?

1. start desktop and see the tabs work as before

- [x] Tests are covering the bug fix or the new feature

### CODE REVIEW

File: packages/ui/src/lib/tab/Tab.svelte [hunk 1]
```diff
@@ -1,21 +1,20 @@
 <script lang="ts">
-import { router } from 'tinro';
-
+export let routerPath: string;
 export let url: string;
```
Comment (feloy) [inline packages/ui/src/lib/tab/Tab.svelte]: I think we should extract the route logic from the component, and have only a `selected` parameter for the tab.

Comment (lstocchi) [reply packages/ui/src/lib/tab/Tab.svelte]: Ok, updating

**Comment by lstocchi** [comment]
Updated the PR to use a selected prop on the Tab component. Not sure if there is a better way to do it (i'm open to suggestions) but i'm not a big fan of this change. I kept the commits separated so you can see the difference. 
Let me know

**Comment by feloy** [comment]
>> I think we should extract the route logic from the component, and have only a selected parameter for the tab.

> Updated the PR to use a selected prop on the Tab component. Not sure if there is a better way to do it (i'm open to suggestions) but i'm not a big fan of this change. I kept the commits separated so you can see the difference. Let me know

Personally, I think that this component should be splitted in two parts:
- the part about router and url should be part of Podman Desktop, as this logic is very specific to the app, but is not a UI concern
- the graphical part should be part of the ui package. And it seems that the `Button type="tab"` could be reused for this part

@deboer-tim @benoitf WDYT?

**Comment by benoitf** [comment]
I'm unsure about the pros/cons about using `Button type="tab"` or `Tab`

like should me merge Tab into button or Button with type tab should not be a button ?

else, to me PR is OK as at least we're getting rid of tinro which is the biggest concern for now

**Review by benoitf — APPROVED** [review]

**Review by feloy — APPROVED** [review]

**Pull request merged**